install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -1,3 +1,45 @@
+{{- define "liveness_probe_args" }}
```

Comment:
I'm not sure this pattern is the easiest to read - the defines are not reused and out of context.
For consistency with other templates I would move them in-line.